docs/design.md

```diff
@@ -577,9 +569,9 @@ The following list includes the major features that are discussed in this docume
 
 - ✅ Inline Playbook
 - ✅ Remote Role
-- ❎ Remote Playbook
+- ✅ Remote Playbook
```

Comment:
I think we did not implemented `Remote Playbook` yet.

Reply:
Yeah, that's a typo. Fixed